### ISSUE TITLE
Stored bid response not found

### ISSUE THREAD

**Issue by VeronikaSolovei9** [prebid/prebid-server#2487]
What is a correct way to handle impression with stored bid response if stored bid response is not found? 
Currently we don't fail request if stored bid response is not found for one or more imps. Those impressions will have no bids back. 
Question: should we run impression with stored bid response as usual impression if stored bid response is not found? 

Follow up for proposed changes in this PR: https://github.com/prebid/prebid-server/pull/2473

**Comment by bretg** [comment]
Hi @VeronikaSolovei9 -- PBS-Java bails on the request with an error. I would suggest the same for PBS-Go

```
Invalid request format: Failed to fetch stored auction response for impId = a and storedAuctionResponse id = 1001-1.
```

**Comment by VeronikaSolovei9** [comment]
Thank you for the answer Bret! If so then we will need to change current behavior and fail the request.

**Comment by VeronikaSolovei9** [comment]
Please do not close this issue until this is fixed.

**Comment by SyntaxNode** [comment]
Implemented in PBS-Go 0.251.0.
